oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elasticsearch/query/ElasticsearchIndex.java

```diff
@@ -90,7 +90,7 @@ protected IndexNode acquireIndexNode(String indexPath) {
 
     @Override
     protected String getFulltextRequestString(IndexPlan plan, IndexNode indexNode) {
-        return Strings.toString(ElasticsearchResultRowIterator.getESRequest(plan, getPlanResult(plan)));
+        return Strings.toString(ElasticsearchResultRowIterator.getESQuery(plan, getPlanResult(plan)));
```

Comment:
Could you name it "getElasticQuery"? I understand "ES" is shorter than "Elasticsearch"... but clearness is more important than saving a bit of length in my view.

Reply:
Will be considered in a separate issue.